### ISSUE TITLE
Feature request: Share to iCal (.ics) files

### ISSUE THREAD

**Issue by BobJonkman** [GetTogetherComm/GetTogether#114]
The only options to share an event, or a team's calendar are to share with LinkedIn or Facebook. I would like to display my team's events in my own Thunderbird/Lightning calendar, or Google calendar. Having an iCalendar sharing option would provide this integration.

.ics files should be available for all events for a team, but also on a single event.

Eventually I would like to see CalDAV functionality, so that I can submit or edit events from my preferred calendar client app.

--Bob.

**Comment by BobJonkman** [comment]
A guest speaker who is not a regular member of the team may want to add just the one event to their calendar; I would provide a link to only one event in the meeting minutes...

I finally found the iCal link :-) but it doesn't show all the events (past and future), only the upcoming event. On KWVoIP there is a single, recurring event, so I thought the iCal feed would have recurrence info in it...

Thanx!
--Bob.

**Comment by mhall119** [comment]
Yeah, the ical feed pre-dates the addition of recurring events in GetTogether. Can you file a separate bug to add them (and past events too, if you think that would be useful)?
